packages/address-utils/package.json

```diff
@@ -7,15 +7,15 @@
   "exports": {
     "./display": {
       "types": "./dist/display.d.ts",
-      "import": "./dist/display.mjs",
+      "import": "./dist/display.js",
       "require": "./dist/display.cjs"
     }
   },
   "files": [
     "dist"
   ],
   "scripts": {
-    "dev": "tsup --watch",
+    "dev": "tsc --watch",
```

Comment:
`tsc` simply compiles the typescript files and outputs `.js` esm modules. I don't know why, but VSCode plays way better with this configuration. It allows you to get proper types and also click on a symbol to go the the typescript source, even if that source is in another monorepo package. We had lost that ability with the previous configuration that used `tsup` in dev mode.